scripts/modules/prompts.js

```
@@ -0,0 +1,432 @@
+		if (isNaN(Number(input))) {
+			return 'Please enter a valid number.';
+		}
+		return true;
+	},
+
+	/**
+	 * Validates that input is a positive integer
+	 */
+	positiveInteger: (input) => {
+		const num = Number(input);
+		if (isNaN(num) || num <= 0 || !Number.isInteger(num)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)